src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -43,10 +46,10 @@ public Future<Integer> loadData(TenantAttributes attributes, String tenantId, Ma
     buildDataLoadingParameters(attributes, tl);
 
     return Future.succeededFuture()
+      // migrationModule value it the same as fromModuleVersion from schema.json
+      .compose(v -> migration(attributes, "mod-invoice-storage-5.7.0", () -> populateInvoiceWithFiscalYear(headers, vertxContext)))
```

Comment:
This mod-invoice-storage-5.7.0 is the same with from attribute based on example pr, but we can disscuss in case of suggestions
![telegram-cloud-photo-size-2-5406968577459869038-y](https://github.com/folio-org/mod-invoice-storage/assets/113523904/7fb00445-cb5c-44a1-8582-dda052c4847d)